webuild-drafts/IntermediaryTesting.md

```diff
@@ -0,0 +1,538 @@
+# 1         Abstract
+
+Relying Party Intermediaries serve as trusted intermediaries between Relying Parties and Wallet Holders, simplifying the integration of EUDIW capabilities while maintaining security and trust guarantees.
+
+There are a number of critical technical challenges for RPI implementation, particularly around dual identification (distinguishing between the intermediary and the actual RP) and embedded disclosure policy evaluation. In order to ensure that the EUDIW implementations will allow for Intermediaries to play their role, Intermediaries should be tested in WE BUILD. These challenges require availability of certain infrastructure components and testing scenarios within the WE BUILD large scale pilot.
+
+# 2         Background
+
+## 2.1       Context
+
+eIDAS 2 [1] explicitly recognizes the role of Relying Party Intermediaries in Article 5b 10: “Intermediaries acting on behalf of relying parties shall be deemed to be relying parties and shall not store data about the content of the transaction”. This role is also acknowledged in the ARF [i.2] and further detailed in sections 3.11.3 and 6.6.5.
+
+## 2.2       The RP Intermediary Role
+
+As defined in draft ETSI TR 119 479-1, an RP Intermediary is:
```

Comment:
I would not rely a technical reports for the definitions of the main actors, don't we have a specification instead of a technical report?